tests/BatchQueueTest.php

```
@@ -88,13 +89,67 @@ public function test_push_includes_container_overrides_when_job_supports_overrid
+
+        $this->database->shouldReceive('table')->with('table')->andReturn($query = m::mock('StdClass'));
+        $query->shouldReceive('insertGetId')->once()->andReturn(9);
+
+        $this->batch->shouldReceive('submitJob')->once()->andReturnUsing(function ($payload) use ($ecsOverrides) {
+            $this->assertArrayHasKey('ecsPropertiesOverride', $payload);
+            $this->assertArrayNotHasKey('containerOverrides', $payload);
+            $this->assertSame($ecsOverrides, $payload['ecsPropertiesOverride']);
+        });
+
+        $this->queue->push(new TestJobWithBothOverrides($ecsOverrides));
+    }
```

PHPMD (2.15.0)

[error] The method test_push_ecs_override_takes_precedence_over_container_overrides is not named in camelCase. (undefined)

(CamelCaseMethodName)